esp8266-fastled-webserver/Map.cpp

```diff
@@ -442,9 +442,9 @@ void drawAnalogClock() {
     float minute = timeClient.getMinutes() + (second / 60.0);
     float hour   = timeClient.getHours()   + (minute / 60.0);
 
-    hourAngle   = 256u - hour   * degreesPerHour;
```

Comment:
OK, rotate 90 degrees.  Math still works because result is stored in `uint8_t`, so it's implicitly modulo 256.  👍